src/client/windshaft.js

```diff
@@ -469,6 +475,9 @@ export default class Windshaft {
             t.categoryNames.map(name => categoryIDs[name] = this._getCategoryIDFromString(name));
             columns.push(t);
         });
+        // FIXME: force columns to include `cartodb_id` in the Metadata
```

Comment:
I thought this wasn't needed after all

Reply:
Yep, it seems that, with the changes in MNS, this is not required. I'll remove it.